### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by paultyng**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+Include any relevant Terraform configuration or project structure:
+
+```terraform
+resource "github_repository" "test" {
+  name = "vscode-terraform"
+}
+
+# etc...
+```
+
```
Comment (radeksimko) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]:
```suggestion
You can use `tree` to output ASCII-based hierarchy of your project:

```
$ tree .
.
|-- main.tf
|-- submodule
|   `-- main.tf
`-- terraform.tfvars

1 directory, 4 files
```
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+name: Feature request
+about: Suggest a new feature or other enhancement.
+labels: enhancement
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (aeschright) [inline .github/ISSUE_TEMPLATE/feature_request.md:8]: Could we link to an example of a good feature request to emphasize what "clear and concise" is?

Comment (aeschright) [reply .github/ISSUE_TEMPLATE/feature_request.md:8]: Doesn't have to be from this project, we could choose one of the providers.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
```
Comment (aeschright) [inline .github/ISSUE_TEMPLATE/feature_request.md:14]: A little repetitive.

**Pull request merged**